force-app/main/default/classes/TestSalesforceAccountFromIntegration.cls

```diff
@@ -0,0 +1,102 @@
+@isTest
+public class TestSalesforceAccountFromIntegration {
```

Comment:
when you start with word TESt you make it as verb. it is fine for methods but not for classes